include/eve/traits/overload/default_behaviors.hpp

```diff
@@ -93,18 +93,91 @@ namespace eve
     friend constexpr bool operator==(callable const&, callable const&)                { return true;  }
   };
 
+  template< template<typename> class Func
+  , typename OptionsValues
+  , typename... Options
+  >
+  struct relative_conditional_callable : callable<Func, OptionsValues, relative_conditional_option, Options...>
+  {
+    using base_t = callable<Func, OptionsValues, relative_conditional_option, Options...>;
+    using func_t =  Func<OptionsValues>;
+
+    struct ignore { template<typename T> operator T() { return T{}; } };
+
+    template<callable_options O, typename T, typename... Ts>
+    constexpr EVE_FORCEINLINE auto behavior(auto arch, O const& opts, T x0, Ts... xs) const
+    {
+      if constexpr (match_option<condition_key, O, ignore_none_>)
+      {
+        return base_t::behavior(arch, opts, x0, xs...);
+      }
```

Comment:
If I understand you correctly, I usually prefer the other way around.
I add `ignore_none` when no ignore is passed.